sql/changes/1.8/constrain-transaction-tables.sql.checks.pl

```diff
@@ -352,11 +352,11 @@ package _18_upgrade_checks;
 the database anymore. As a result, the files attached to these transactions
 cannot remain connected. Two options are available:
 
-1. 'Delete' the files listed below  
```

Comment:
This whitespace was there with intent: it forces a new line within the same paragraph. Could you undo the removal, please?

Reply:
Fixed.